examples/hosts/app-integration/external-data/src/mock-service-interface/dataParsing.ts

```diff
@@ -24,14 +30,13 @@ export interface ParsedTaskData {
  *
  * TODO: See notes below about moving away from plain string to something more realistic.
  */
-export function parseStringData(stringData: string): ParsedTaskData[] {
-    const taskStrings = stringData.split("\n");
-    return taskStrings.map((taskString) => {
-        const [taskIdString, taskNameString, taskPriorityString] = taskString.split(":");
+export function parseTaskData(taskData: object): ParsedTaskData[] {
```

Comment:
Do we need this and ParsedTaskData anymore? Or can we just operate on `TaskData` directly?